src/server.c

```
@@ -5876,7 +5928,7 @@ int main(int argc, char **argv) {
     readOOMScoreAdj();
     initServer();
     if (background || server.pidfile) createPidFile();
-    redisSetProcTitle(argv[0]);
+    redisSetProcTitle(NULL);
```

Can we assert here then, and not silently ignore the error code?